rmgpy/species.py

```diff
@@ -162,17 +162,17 @@ def setMolecularWeight(self, value):
         self._molecularWeight = quantity.Mass(value)
     molecularWeight = property(getMolecularWeight, setMolecularWeight, """The molecular weight of the species. (Note: value_si is in kg/molecule not kg/mole)""")
 
-    def generateResonanceIsomers(self, keepIsomorphic=True):
+    def generate_resonance_structures(self, keepIsomorphic=True):
         """
```

Comment:
this is beautiful! now I won't have to look up to determine whether structures or isomers is used!